components/LinkAnchor/LinkAnchor.tsx

```diff
@@ -0,0 +1,34 @@
+import type { LinkProps } from 'next/link';
+import Link from 'next/link';
+import type { AnchorHTMLAttributes } from 'react';
+import { forwardRef, memo } from 'react';
+
+type LinkAnchorProps = LinkProps & AnchorHTMLAttributes<HTMLAnchorElement>;
+
+export const LinkAnchor = memo<LinkAnchorProps>(
```

Comment:
Chyba już niepotrzebne?
```suggestion
export const LinkAnchor = memo(
```